modules/valkey/main.tf

```diff
@@ -21,7 +21,7 @@ resource "google_memorystore_instance" "valkey_cluster" {
   engine_version = var.engine_version
   mode           = var.mode
 
-  desired_psc_auto_connections {
+  desired_auto_created_endpoints {
```

Comment:
@bai this change will recreate all the instances that were created with `desired_psc_auto_connections`. We will think about how to remove desired_psc_auto_connections in future version.